jet-fuel/app/models/url.rb

```diff
@@ -1,6 +1,7 @@
 class Url < ActiveRecord::Base
   before_validation :shorten_url
-  validates :full_url, presence: true, uniqueness: true
+  validates :full_url, presence: true, uniqueness: true,
+    format:   URI::regexp(%w(http https))
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Unnecessary spacing detected.